editor/src/components/inspector/inspector.tsx

```diff
@@ -714,6 +720,34 @@ export const InspectorContextProvider = React.memo<{
   const onUnsetValue = React.useCallback(
     (property: PropertyPath | Array<PropertyPath>, transient: boolean) => {
       let actionsArray: Array<EditorAction> = []
+
+      const propTop = PP.create('style', 'top')
+      const propBottom = PP.create('style', 'bottom')
+      const propLeft = PP.create('style', 'left')
+      const propRight = PP.create('style', 'right')
+
+      const maybeInvalidGroupState = maybeInvalidGroupStates(
```

Comment:
Minor: I found it a little confusing that these are so similarly named.

Reply:
https://github.com/concrete-utopia/utopia/pull/3977/commits/38d544f75a43228f7b6a4877e84247ead37cde2f